### PR TITLE
Some small improvements and formatting

### PR DESCRIPTION
**Pull request by hamidalakbarov**
Some small improvements and formatting

### CODE REVIEW

File: configuration.properties [hunk 1]
```diff
@@ -2,7 +2,7 @@ browser=chrome
 url=https://qa.meetsky.net
 username=User1
 password=Userpass123
-filePathUS_09=/Users/hamidalakbarov/Desktop/File to upolad today.txt
+filePath=/Users/hamidalakbarov/Desktop/File to upolad today.txt
```
Comment (hamidalakbarov) [inline configuration.properties:5]: Everybody can provide its own path here

Comment (Sayora1) [reply configuration.properties:5]: sounds good!

**Review by latifaforozan — APPROVED** [review]
sounds good Hamid, Cant wait to run with new implementation.

**Review by SForogh — APPROVED** [review]
Looks perfect! And 'beautified' :)

**Pull request merged**